.jenkins/tests.Jenkinsfile

```diff
@@ -0,0 +1,185 @@
+/* A Jenkins pipeline that will handle PR and nightly tests
+*  These are the original pipelines:
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-makefile.yml
```

Comment:
Should this PR be removing the azure pipelines ".yml" files? if yes then you can remove this comment, if no then maybe whenever we do completely remove them we will need to update this comment

Reply:
We should run both the ADO and nightly pipeline together for 10 days to establish there is parity between the two.

Reply:
I'm thinking we could wait until we completely finish the migration over to Jenkins at least.